modules/microshift-cni-multus-bridge-example-config.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-bridge-example-config_{context}"]
+= Example configuration for the bridge additional network
+
+Use the following procedure to configure a bridge-type additional network. You can use the example workflow here to configure other supported additional network types.
+
+[IMPORTANT]
+====
+The minor version of {microshift-short} and the version of the {microshift-short} Multus CNI plugin must match. You must use the 4.16 version of both RPMs, for example.
+====
```

Comment:
Is it even possible? I would expect the multus-rpm to have an exact match dependency on the MicroShift base package. Thus, it would be impossible to install bad combos. So we might be able to remove this warning.
If not, I am wondering if it belongs to an Example section. Shouldnt it be in the "Install multus" section?

Reply:
@pmtk WDYT? Could a user potentially try to install Multus 4.16 on an earlier MicroShift as a Day 2 operation? Or can we remove the admonition?